ush/sample_config/ARL_run/config.yaml_cold_aqmna13km_1day_wcoss2

```diff
@@ -82,5 +82,6 @@ cpl_aqm_parm:
   NEXUS_INPUT_DIR: /lfs/h2/emc/lam/noscrub/RRFS_CMAQ/nexus_emissions
   NEXUS_FIX_DIR: /lfs/h2/emc/lam/noscrub/RRFS_CMAQ/nexus/fix
   NEXUS_GRID_FN: grid_spec_793.nc
+  RUN_TASK_NEXUS_GFS_SFC: true
```

Comment:
@bbakernoaa , this is set on Line 55 above, please remove this line

Reply:
why are we turning this off?

Reply:
ok